### PR TITLE
fixed layout issue on grid grouped people layout

### PR DESCRIPTION
**Pull request by josephgknox**

### CODE REVIEW

File: css/stanford_jumpstart_engineering.css [hunk 1]
```diff
@@ -51,4 +51,10 @@
   .style-custom-styles-rich #block-bean-jse-linked-logo-block p a {
     padding-bottom: 10px;
   }
+}
+
+/* SOE footer logo for cardinal style */
+
+.views-grid-four .views-row.views-row-even:nth-of-type(4n) {
+    margin-right: 0;
 }
```
Comment (cjwest) [inline css/stanford_jumpstart_engineering.css]: @josephgknox: It looks like you're missing a carriage return at the end of the file.

**Comment by josephgknox** [comment]
Thanks, Caryl! I've also updated the styles entirely (which was necessary after further testing).

**Comment by cjwest** [comment]
Yeah! It's fixed!

File: css/stanford_jumpstart_engineering.css [hunk 2]
```diff
@@ -51,4 +51,37 @@
...
+.views-grid-four .views-row.views-row-76,
+.views-grid-four .views-row.views-row-80,
+.views-grid-four .views-row.views-row-84,
+.views-grid-four .views-row.views-row-88,
+.views-grid-four .views-row.views-row-92,
+.views-grid-four .views-row.views-row-96,
+.views-grid-four .views-row.views-row-100,
+.views-grid-four .views-row.views-row-104,
+.views-grid-four .views-row.views-row-108,
+.views-grid-four .views-row.views-row-112 {
```
Comment (sherakama) [inline css/stanford_jumpstart_engineering.css:85]: What happens after row item 112?

Comment (josephgknox) [inline css/stanford_jumpstart_engineering.css:85]: When the 116th item is added to a grouping, the layout would flow improperly (previously, this was only styled for 28 items). We can make this as big as needed.

I'm confident that there is a way to address this with the adjacent and/or nth-child CSS selectors. After trying several equations, however, I was unable to get this reduced down to a few lines of code (as the view's groupings breaks the standard nth-child convention - I tried to target every fourth item, but it recalculates with each grouping).

I filed a big in OFW, which is where this code lives. Brian poked at it for a bit, too. No luck... yet. If and when we have a more succinct fix, I'll add it to OFW and SOE.

Due to the timeliness of this, I thought adding the above fix was the best route to take now.

Let me know your thoughts.

Comment (jbickar) [inline css/stanford_jumpstart_engineering.css:85]: @jbickar likes this

Comment (josephgknox) [inline css/stanford_jumpstart_engineering.css:85]: @jbickar @sherakama :o)

Comment (sherakama) [inline css/stanford_jumpstart_engineering.css:85]: Code like this screams at me "there must be a better way"

To reproduce this output exactly without writing as much code I would suggest using using a for loop and sass.
http://thesassway.com/intermediate/if-for-each-while

"Every 4th item" sounds like something that could be accomplished with css3. I'll ponder this.

Comment (josephgknox) [inline css/stanford_jumpstart_engineering.css:85]: Thanks, @sherakama! I'll look into the SASS option. Unfortunately, the CSS nth-child(4n) won't work smoothly, as each grouping in the view calculates slightly differently depending on how many items are in the previously grouping (this was my first hunch, too, but was unable to get it working). I'll continue to poke.

Comment (sherakama) [inline css/stanford_jumpstart_engineering.css:85]: sounds like your solution may not be in the css but rather in the output.

Comment (josephgknox) [inline css/stanford_jumpstart_engineering.css:85]: ...of the view?

Comment (sherakama) [inline css/stanford_jumpstart_engineering.css:85]: Yeah, wrapping each section in a div would make the nth-of-type so much easier.

Alternatively you could do something like this:

`
.views-grid-four h2 + div ~ div:nth-of-type(4n) {
   background: #F0F;
}
`

Comment (sherakama) [inline css/stanford_jumpstart_engineering.css:85]: You will have to browser test that one tho.

**Pull request merged**